runtime/parachains/src/reward_points.rs

```diff
@@ -22,42 +22,71 @@
 //! for the time being, although we will build schemes to do so in the future.
 
 use crate::{session_info, shared};
-use frame_support::traits::ValidatorSet;
-use primitives::v2::ValidatorIndex;
+use frame_support::traits::{Defensive, ValidatorSet};
+use primitives::v2::{SessionIndex, ValidatorIndex};
+use sp_std::collections::btree_set::BTreeSet;
 
 /// The amount of era points given by backing a candidate that is included.
 pub const BACKING_POINTS: u32 = 20;
+/// The amount of era points given by dispute voting on a candidate.
+pub const DISPUTE_STATEMENT_POINTS: u32 = 20;
```

Comment:
are these numbers made up or sensible in some way?

How many era points do people get on average? and compared to making blocks and being online and whatever?